lib/hooks/user.ts

```diff
@@ -2,12 +2,18 @@ import { User } from ".prisma/client";
 import axios from "axios";
 import { useRouter } from "next/dist/client/router";
 import { useEffect } from "react";
-import { useMutation, useQuery, UseQueryResult } from "react-query";
+import { useMutation, useQuery } from "react-query";
```

Comment:
Is there a reason why useMutation imported? Don’t think it’s being used. You can remove it later if you would like to.

Reply:
https://github.com/calebmech/Book-Bazar/blob/44bf81b4fa07c2906878a01239b27be40872f1c4/lib/hooks/user.ts#L19-L36

Reply:
Ahh I see, sorry lol.